src/cardsavr/CardsavrHelper.ts

```diff
@@ -138,7 +138,7 @@ export class CardsavrHelper {
                         {cardholder_id: login.user_id,
                         username: merchant_creds.username, 
                         password: merchant_creds.password, 
-                        site_hostname: merchant_creds.site_hostname}, login.cardholder_safe_key ),
+                        merchant_site_id: merchant_creds.merchant_site_id}, login.cardholder_safe_key ),
```

Comment:
@squann I thought we were still going to require hostname when posting a job.

Reply:
No, it was specifically requested to be removed.

Reply:
Its still in the logging, if thats what you mean? Or it will be anyway.

Reply:
@markbudos